src/features/communication/communication.tsx

```diff
@@ -0,0 +1,46 @@
+import { Paragraph, Subtitle, Title } from '@/app/components';
+import { DiscordLogo } from '@/icons/discord-logo';
+import {
+  CourseNames,
+  RS_DOCS_COMMUNICATION_LINK,
+  discordLinks,
+} from '@/shared/data/communication.data';
+
+import './communication.scss';
+
+interface RequiredProps {
+  courseName: CourseNames;
+}
+
+export const Communication = ({ courseName }: RequiredProps) => {
+  return (
+    <section className="communication container">
+      <article className="communication content info-wrapper">
+        <Title text="Communication" hasAsterisk />
+        <div className="column-2">
+          <figure className="disclogo-wrapper">
+            <DiscordLogo />
+          </figure>
+          <div>
+            <Subtitle text="Discord is the main communication channel in RS School" />
+            <Paragraph>
+              Here is link for the{' '}
+              <a href={discordLinks[courseName]} target="_blank" className="communication-link">
+                course discord server
+              </a>{' '}
```

Comment:
Please make links to look like links, add underscore or external link icon, so user at the first look understand that this is a link and don't need to hover over them to realize that

![image](https://github.com/rolling-scopes/site/assets/114234698/76a8314e-7d89-47e4-a5ad-bd4174ebb6e3)
or
![image](https://github.com/rolling-scopes/site/assets/114234698/3354dfa0-27c1-4d18-b18e-dfc39bd69284)

Reply:
@Quiddlee, That's a great suggestion, but I'd say we should create a separate issue for that

Reply:
I suggest to create smth like `LinkExternal` component